### ISSUE TITLE
Nullish Coalescing in version 5.0.3

### ISSUE THREAD

**Issue by rs7** [WebHotelier/webpack-fast-refresh#6]
After update to 5.0.3 I had a problem

```
/home/rs7/work/calypso/node_modules/@webhotelier/webpack-fast-refresh/index.js:68
    const webpackMajorVersion = parseInt(version ?? '', 10);
                                                  ^

SyntaxError: Unexpected token '?'
```

Can you put `.babelrc` to you code? :)

**Comment by apostolos** [comment]
Thanks for the report.

Fixed in 2e192fa352cbc1ad056d7c4d6486efd0783527a9. Please use v5.0.4

**Comment by rs7** [comment]
Thank you :)
